client/dive-common/components/Viewer.vue

```diff
@@ -310,6 +311,7 @@ export default defineComponent({
           populateConfidenceFilters(meta.confidenceFilters);
           datasetName.value = meta.name;
           frameRate.value = meta.fps;
+          originalFps.value = meta.originalFps;
```

Comment:
If this isn't available it needs to default to a reasonable value, I guess the base fps might be the best?

Reply:
No, if this isn't available I'm going to fall back to the old behavior.  Need to update the PR.